skymp5-server/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "scripts": {
     "build-ts": "tsc",
-    "cmake-js-fetch": "cmake-js configure --directory cmake-js-fetch --out cmake-js-fetch-build"
+    "cmake-js-fetch": "cmake-js configure -G \"Visual Studio 16 2019\" --directory cmake-js-fetch --out cmake-js-fetch-build"
```

Comment:
Yeah, we need Linux support, but it's simple to work around here:

1. mod `package.json` a bit
```json5
 "scripts": {
// ...
    "cmake-js-fetch": "cmake-js configure --directory cmake-js-fetch --out cmake-js-fetch-build"
    "cmake-js-fetch-msvc": "cmake-js configure -G \"Visual Studio 16 2019\" --directory cmake-js-fetch --out cmake-js-fetch-build"
// ...
```

2. Add a condition here `if(MSVC)  cmake-js-fetch-msvc else() ...`
https://github.com/skyrim-multiplayer/skymp/blob/b0420f201ce9cbbaf3df1965b69044b1670d816b/skymp5-server/CMakeLists.txt#L27